radar/data_management/github.yaml

```diff
@@ -0,0 +1,10 @@
+version: 1
+name: Github
+blip:
+  - version: "1.0"
+    ring: ADOPT
+description: |
+  Github Gist allows GitHub users to share pieces of code or other notes.
```

Comment:
Seems to be describing GitHub Gist instead of GitHub

Reply:
Rewrote the description